python/cuml/dask/common/dask_df_utils.py

```diff
@@ -79,3 +81,29 @@ def to_dask_cudf(futures):
     dfs = [d for d in futures if d.type != type(None)]  # NOQA
     meta = c.submit(get_meta, dfs[0]).result()
     return dd.from_delayed(dfs, meta=meta)
+
+
+def to_dask_df(dask_cudf):
+    """
+    Convert a Dask-cuDF into a Pandas-backed Dask Dataframe.
+
+    :param dask_cudf : dask_cudf.DataFrame
+    :return : dask.DataFrame
+    """
+
+    def to_pandas(df, r):
+        return df.to_pandas()
+
+    c = default_client()
```

Comment:
I would be happier if we had a rule that whenever we need a client, we allow the user to pass it in and use default_client as a fallback, since that allows user flexibility... wdyt?

Reply:
Fair enough. I can do that.

Reply:
I've been going back and forth about this particular design detail for some time now, actually. In the beginning, I was accepting Dask `Client` instances everywhere. I even asked Divye to design PR #916 with that in mind.

I've since realized accepting the client in the public API isn't going to make our Dask code more flexible when Dask's own API is using `default_client` everywhere. From what I've seen, Dask-ML doesn't accept a client anywhere in the public API.

The fact that Dask and Dask-ML implicitly assume the `Client` make me think that users are only expected to be interacting with a single `Client` in each process.